File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F15.P2.kt
// BEGIN PROBE F15.P2
fun <T : Comparable<T>> maxOf(a: T, b: T): T = if (a > b) a else b

fun probe(): Int {
    val m = maxOf(3, 5)
    return m
}
// END PROBE F15.P2

fun main() {
    println(probe())
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/swift/F15.P2.swift
func greater() -> Int32 {
// BEGIN PROBE F15.P2
func maxOf<T: Comparable>(_ x: T, _ y: T) -> T { x > y ? x : y }
let m: Int32 = maxOf(3, 5)
return m
// END PROBE F15.P2
}

print(greater())
